src/amitaRJun20/ReplaceSecondLastStringInarray.java

```diff
@@ -0,0 +1,46 @@
+/*Assignement 30 : 18th July 2020
+ 
+Replace second last globant with Technocredits. 
+input-> Hi Globant Hello Globant Hi Globant GM Globant Hi Globant
+
+output->  Hi Globant Hello Globant Hi Globant GM Technocredits Hi Globant*/
+package amitaRJun20;
+
+public class ReplaceSecondLastStringInarray {
+	
+	String replacedSecondLastString(String input, String replaceWord, String replacedWithWord) {
+		int newLength = input.replaceAll(replaceWord, "").length();
+		int totalCount = (input.length() - newLength) / replaceWord.length();
+		String outputString = "";
+		int count = 0;
+		String[] strArray = input.split(" ");
+		if(totalCount<2) {
+			System.out.println("The String has no second last " +replaceWord+" word");
+			return input;
+		}
+		for (int index = 0; index < strArray.length; index++) {
```

Comment:
In for you can iterate only 2 times(last and second last). This will save iteration in case multiple occurence.

Reply:
Yes you are correct...Actually i did this to practice the way Maulik has suggested to get the totalCount in the first approach...
Thanks for approving...